tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/AutoCreateExpiryResourcesTest.java

```diff
@@ -59,6 +59,7 @@ public void setUp() throws Exception {
 
       // set common address settings needed for all tests; make sure to use getMatch instead of addMatch in invidual tests or these will be overwritten
       server.getAddressSettingsRepository().addMatch("#", new AddressSettings().setAutoCreateExpiryResources(true).setExpiryAddress(expiryAddress).setExpiryDelay(EXPIRY_DELAY));
+      server.getAddressSettingsRepository().addMatch(String.valueOf(expiryAddress), new AddressSettings().setExpiryDelay(-1L));
```

Comment:
I'm actually curious why it is setting expiry delay on the regular address settings to begin with, and whether that should instead of adding this? 

Most of the tests look to send messages that have expiration and are already-expired, so the setting shouldnt affect them after this change.

The exceptions seem to be testAutoCreatedExpiryFilterAnycast / testAutoCreatedExpiryFilterMulticast that seem to send messages without expiration. Could it just change to setting one, and then remove the ExpiryDelay setting here?

Reply:
There are other tests in this test class that will use this:

https://github.com/apache/activemq-artemis/blob/14c564a48144b0e052c6e0eeee64ba0f94744335/tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/AutoCreateExpiryResourcesTest.java#L136-L138


I will use a prefix rather than everything.

Reply:
It would be a complete rewrite of the test.. I will just go with the extra line.

Reply:
other tests are failing in the same situation if I don't set this..


I had added more loops and more sleeps to other tests and they also failed.


I would rather keep this rather to rewrite this test.

Reply:
I could also keep running them in loop to add individually on every tests needed.. so I would rather just keep it.

Reply:
I will mark it as resolved.. please unmark it if you disagree.

Reply:
Fair enough, you had only mentioned the FQQN one so I thought it was the only one actually needing this (which is why I didnt like changing it for all tests).